ergoCubSN001/calibrators/left_arm-calib.xml

```diff
@@ -31,7 +31,7 @@
 	</group>
 
 <!--	<param name="CALIB_ORDER"> (4 5 6) (3) (2) (0) (1) (7 8) (9) (10 11 12) </param> --> <!-- Don't remove this line -->
-	<param name="CALIB_ORDER">  (4 5 6) (3) (2) (0) (1) (7 8) (9) (10 11 12) </param> 
+	<param name="CALIB_ORDER">  (4 5 6) (3) (2) (0) (1) (7 8) (10 11 12) </param> 
```

Comment:
Hi @S-Dafarra @GiulioRomualdi 
Any specific reason why joint 9 is not included?

Reply:
That is the index abduction. It has been removed by @AntonioConsilvio in https://github.com/robotology/robots-configuration/pull/681/commits/cff4ac3e282bdaea6e94c14ed3ac8028a840837c

@AntonioConsilvio I suppose it is still broken right?

Reply:
Gotcha!